css/main.css

```diff
@@ -38,13 +38,13 @@ h1 {
 }
 
 .hero-unit h1 {
-  color: black;
-  background-color: white;
-  display: inline-block;
-  font-family: 'Zilla Slab Highlight', sans-serif;
-  font-weight: bold;
+  color: white;
+  background-color: black;
+  display: inline;
+  font-family: 'Zilla Slab', sans-serif;
```

Comment:
I'd like to use Highlight as well as possible. You are correct, for the mobile case it's not possible with any of the display properties because of the white background. Could you create an issue with a small testcase in the zilla repo at https://github.com/mozilla/zilla-slab?

As this works perfectly well until the text gets wrapped, do you think it would make sense to only apply this part here for smaller screens?

Reply:
The problem lies with the font itself, i'll create an issue in the respected repo for the same.
As of now we can use media query at the particular break point and all my suggested changes could go there. This way 'Zilla Slab Hightlight' would stay in the desktop view and in mobile/tablet view 'Zilla Slab' would come up.
What do you think about this?

Reply:
Let's go for it.